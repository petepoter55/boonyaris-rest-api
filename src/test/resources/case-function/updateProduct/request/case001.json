{
  "productUpdateRequest": {
    "productCode": "product-000376",
    "productName": "แอปเปิ้ลแดง",
    "productType": "V",
    "description": "",
    "productQuantity": "35",
    "price": "100.55",
    "status": ""
  },
  "product": {
    "id": 1,
    "productCode": "product-000376",
    "productType": "V",
    "productName": "แอปเปิ้ลเขียว",
    "description": "แอปเปิ้ลเขียวนำเข้าจากประเทศนิวซีแลนด์มีรสชาติหวานและเปรี้ยวเล็กน้อย",
    "status": "active",
    "productQuantity": "100",
    "price": "139.00",
    "createBy": null,
    "updateBy": null,
    "createDateTime": "2022-05-12 00:00:00",
    "updateDateTime": "2022-05-12 00:00:00",
    "delFlag": null
  }
}