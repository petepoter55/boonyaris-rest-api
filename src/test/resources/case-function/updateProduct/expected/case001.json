{
  "id": 1,
  "productCode": "product-000376",
  "productType": "V",
  "productName": "แอปเปิ้ลแดง",
  "description": "แอปเปิ้ลเขียวนำเข้าจากประเทศนิวซีแลนด์มีรสชาติหวานและเปรี้ยวเล็กน้อย",
  "status": "active",
  "productQuantity": "100",
  "price": "100.55",
  "createBy": null,
  "updateBy": null,
  "createDateTime": "2022-05-12 00:00:00",
  "updateDateTime": "2022-05-12 00:00:00",
  "delFlag": null
}